doc/ref_arch/kubernetes/chapters/chapter02.rst

```diff
@@ -1408,7 +1408,9 @@ machines or containers.
      - Infrastructure
      - Network
      - The Architecture must support fully redundant network connectivity to the Kubernetes
-       nodes, leveraging multiple network connections.
+       nodes, leveraging at least two physical network connections per node. If the node is
+       virtualised, the redundant network interfaces must be implemented on the Hypervisor
+       host.
```

Comment:
what is a "hypervisor host"?
what is stated here is confusing. Would the following work (see [kubevirt](https://kubevirt.io/user-guide/virtual_machines/interfaces_and_networks/):

The Architecture must support fully redundant network connectivity to the Kubernetes nodes, viz., at least two (2) physical network connections per Kubernetes node. For virtualised nodes, redundant network interfaces, backed by the physical networks, are added to the VM.

Reply:
@pgoyal01 I have updated this in https://github.com/cntt-n/CNTT/pull/3215/commits/9f0a250ff2e5e70be11b92c5ff3d751451a6125e , please let me know if ok, thanks